### DISCUSSION TITLE
Xfinity Stream

### DISCUSSION THREAD

**Discussion by ampolisano** [FrostCo/AdvancedProfanityFilter#490] [Ideas]
Are there any plans to add the muting functionality to the Xfinity Stream? https://www.xfinity.com/stream/

The add-on works with the subtitles, but it does not mute the audio.

**Comment by richardfrost** [comment]
@ampolisano I don't have access to Xfinity right now, but if I can find someone who has it that's willing to help me gather some information I can probably add support for it. I probably won't have time to work on it more for a couple weeks though. Thanks for the suggestion!

**Reply by ampolisano** [reply]
I have it and would be more than happy to assist.  Please let me know what you need once you have time.  And thank you so much for your hard work and efforts on this!

**Reply by richardfrost** [reply]
You're welcome! Thanks for being willing to help out @ampolisano. Unfortunately I'm probably not going to be available until next week, but can you send me an email at apf @ frostco.dev (without the spaces) and we can coordinate the process better.

**Reply by richardfrost** [reply]
@ampolisano, I created an issue with the information on what I have found so far: https://github.com/FrostCo/AdvancedProfanityFilter/issues/495.
